### PR TITLE
Fix for windows and missing include

### PR DESCRIPTION
**Pull request by MichaelKirsch**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -41,6 +41,11 @@ int main()
     Keyboard keyboard;
     sf::Clock updateClock;
 
+    //gui bools
+    bool m_profilerOpen= false;
+    bool m_infoOpen = false;
+    bool m_brushOpen = false;
```
Comment (Hopson97) [inline src/main.cpp:47]: `m_` prefix is only used for class members. However, the bar is not needed (for anyways, wasn't planning to have this epc. for the entire game) 

An alternative could be pressing a key eg ` or `F3` to open the debug/profiling menus

File: src/main.cpp [hunk 2]
```diff
@@ -117,11 +122,30 @@ int main()
...
+                        ImGui::EndMenu();
+                    }
+                    if (ImGui::BeginMenu("Settings"))
+                    {
+                        ImGui::Checkbox("Profiler",&m_profilerOpen);
+                        ImGui::EndMenu();
+                    }
+                    ImGui::EndMainMenuBar();
+                }
+            }
```
Comment (Hopson97) [inline src/main.cpp:142]: See above comment

File: src/Game/ScreenGame.h [hunk 3]
```diff
@@ -40,4 +40,6 @@ class ScreenGame final : public Screen {
     sf::Vector2i m_editPivotPoint;
     sf::Vector2i m_editEndPosition;
     bool m_quadDrag = false;
+
+
```
Comment (Hopson97) [inline src/Game/ScreenGame.h:44]: :eyes:

File: src/Game/Map.h [hunk 4]
```diff
@@ -53,6 +54,7 @@ struct Map {
     void setTile(const sf::Vector2i& position, TileType type);
 
     void draw(sf::RenderWindow* target);
+    void regenerate();
```
Comment (Hopson97) [inline src/Game/Map.h:57]: Not defined, I have a separate function now anyways for this

**Review by Hopson97 — CHANGES_REQUESTED** [review]

**Comment by MichaelKirsch** [comment]
Ill did everything as you wanted

**Review by Hopson97 — APPROVED** [review]

**Pull request merged**